I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang.xml

```diff
@@ -580,32 +641,80 @@ OTS Media Link represents a media link:
+   gain bandwidth, which can be quite flat, with similar or even lower
+   noise figures compared to EDFAs. On the other hand, Raman amplifiers
+   consume more power and are usually more expensive than EDFAs.
+  </t>
+  
+  <t>
+   Raman amplifiers are distributed amplifiers where an optical pump
+   signal is injected typically in opposite direction to the optical
+   signal that is amplified (backward pump, counter-propagating pump
+   light). Injecting the optical pump signal in the same direction is
+   also possible (forward pump, co-propagating pump light).
+   For optical amplifiers, the YANG model defines Raman pump light
+   attributes describing the direction (raman-direction) with respect
+   to the signal that is amplified and optical frequency and power for
+   the pump light source(s) contained in the raman-pump list. These   
+   Raman amplifier-specific attributes are optional as they are only
+   applicable to Raman amplifiers. For determining the optical amplifier
+   type, i.e., to figure out whether an optical amplifier is a Raman
+   amplifier, the type-variety attribute is used.
+   Due to the distributed nature of the Raman amplifier it is difficult
+   to clearly separate the amplifier from the fiber span into which the
+   pump signal is injected. From a topology modeling perspective, the
+   Raman amplifier is modeled as two OMS line elements:
+  </t>
+  
+  <t>
+    <list style="numbers"><?rfc subcompact="yes"?>
+      <t>
+       a passive fiber element accounting for the fiber loss only
+       and not the resulting loss including the Raman gain</t>
+      <t>
+       an amplifier element providing all optical amplifier properties
+       (gain, tilt, etc.). On the OMS-link, the amplifier element is
+       placed where the pump is located and the geolocation information
+       also indicates the location of the pump.
+      </t>
+    <?rfc subcompact="no"?>
+    </list>
+  </t>
+  
+  <t>
+   Amplifiers can be classified according to their location along the
+   TE-link. There are three basic types of amplifiers: ILA,
    Pre-Amplifier and Booster. ILA is In-Line Amplifier which is a
-   separate node type while Pre-Amplifier and Booster Amplifier are
-   integral elements of ROADM node. From a data modeling perspective,
-   Pre-Amplifier and Booster Amplifier are internal functions of a
-   ROADM node and as such these elements are hidden within ROADM node.
-   In this document, we would avoid internal node details, but attempt
-   to abstract as much as possible.
+   separate physical device while Pre-Amplifier and Booster Amplifier
+   are integral elements of a WDM-TE-node. From a data modeling
+   perspective, Pre-Amplifier and Booster Amplifier are internal
+   functions of a WDM-TE-node and as such these elements are hidden
+   within WDM-TE-node. In this document, we would avoid internal node
+   details, but attempt to abstract as much as possible.
```

Comment:
As far as I have understood, the BA and PA are part of the WDM node (physical device) but not of the WDM-TE node, but instead, of the TE Link (as OMS elements)

Reply:
Modified text:
   Amplifiers can be classified according to their location along the
   TE-link (OMS MCG). There are three basic amplifier types: In-Line
   Amplifiers, Pre-Amplifiers and Booster Amplifiers. ILAs are separate
   physical devices while Pre-Amplifiers and Booster Amplifiers are
   integral elements of a WDM-node. From a data modeling perspective,
   node-internal details should not be modeled and should be abstracted
   as much as possible. For Pre-Amplifiers and Booster Amplifiers,
   however, a different approach has been taken and they are modeled as
   TE-link elements as they have the same optical impairments as ILAs.